src/GitHub.Exports/Services/Services.cs

```diff
@@ -141,5 +126,10 @@ public static Repository GetRepoFromSolution(this IVsSolution solution)
                 return null;
             return new Repository(repoPath);
         }
+        static UriString GetUri(IRepository repo)
```

Comment:
This is the same logic as IGitService.GetUri(repo), but since this is only called from GetRepoFromSolution (which is static), I'm just leaving this bit here because it's a one liner and faster than looking up the service.